argocd/overlays/moc-infra/applications/envs/moc/smaug/operate-first/connaisseur-application.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
we also need to include this file path in here
https://github.com/operate-first/apps/blob/master/argocd/overlays/moc-infra/applications/envs/moc/smaug/operate-first/kustomization.yaml
for argocd to deploy the app.